fiftyone/utils/ultralytics.py

```
@@ -701,6 +701,129 @@ def _predict_all(self, imgs):
+        prompts = [item.pop("prompt", None) for item in batch]
+        orig_images = [img.get("orig_img") for img in batch]
+        orig_shapes = [_get_image_dims(img)[::-1] for img in orig_images]
+        images = torch.stack([img.get("img") for img in batch])
+        return {
+            "orig_imgs": orig_images,
+            "images": images,
+            "orig_shapes": orig_shapes,
+            "prompts": prompts,
+        }
+
+    def _predict_all(self, imgs):
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `_predict_all`

(ANN202)